### ISSUE TITLE
Windows Upload File Fail.

### ISSUE THREAD

**Issue by OloBo-MSK** [RocketChat/Rocket.Chat.Electron#1577]
## My Setup

- Operating System: Windows 10 Pro
- App Version: 2.17.9
- RocketChat server: tested on 3.0.10 and 3.1.0
- Installation type: <exe>

- [x] I have tested with the latest version
- [x] I can simulate the issue easily

## Description
Can't send (ZIP) file when Accepted Media Types value is: application/zip 
If i change value to application/* everything works fine.
From any browser on PC => same problem.

P.S.
From MacOS client everything is ok.

### Current Behavior
When application/zip => Error: File type is not accepted. [error-invalid-file-type]
When application/* => File sent.

**Comment by havx** [comment]
We're experiencing this same issue in our environment. Only occurs on Windows in both the desktop client and the web browser. I've determined this issue affects zip, 7z, bz2, and gz files.

OS: Windows 10 Enterprise
App version: 2.17.1 through 2.17.9
Browser: Firefox 76.0.1; Chrome 83.1.4103.61; Microsoft Edge 83.0.478.37
Server version: 3.0.12

Looking at the HAR log for the postData in the browsers, each appear to be sending the wrong content type. All browsers for each of those file types send `Content-Type: application/octet-stream` instead of the correct types: `application/zip,application/x-7z-compressed,application/x-bzip2,application/gzip`.

The response generated in the browser and client is "bad request" and the error in the HAR log is: ` "{\"success\":false,\"error\":\"File type is not accepted. [error-invalid-file-type]\",\"errorType\":\"error-invalid-file-type\"}"`

**Comment by sematro** [comment]
The same error occurs with us. We have noticed it with MS Office files.

It is so that the browser asks the operating system for the mimetype of certain files. 
E.g. described here: [https://developer.mozilla.org/en-US/docs/Mozilla/How_Mozilla_determines_MIME_Types#ExternalHelperAppService](https://developer.mozilla.org/en-US/docs/Mozilla/How_Mozilla_determines_MIME_Types#ExternalHelperAppService)

Under Windows, this information is taken from the registry.

If Microsoft Office is installed, the Mimetype (Content Type) is set correctly there. 
But since we use Libreoffice, this was not the case. 

To fix this, we now set a REG_SZ entry called `Content Type` with the value `application/vnd.openxmlformats-officedocument.spreadsheetml.sheet` in `HKEY_CLASSES_ROOT\.xlsx` for example.
The mimetype is then transferred correctly to the browser.

I don't know exactly who is making the mistake. It only becomes a problem if you block `application/octet-stream`.
